pkgs/by-name/ka/karlender/package.nix

```diff
@@ -0,0 +1,92 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitLab,
+  pkg-config,
+  gtk4,
+  libadwaita,
+  wrapGAppsHook4,
+  glib,
+  tzdata,
+  nix-update-script,
+  dbus,
+  cargo-gra,
+  fetchpatch,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "karlender";
+  version = "0.10.4";
+
+  src = fetchFromGitLab {
+    owner = "floers";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-W+s1RCPwy7ZiK514AaQLwV9y+VJ58oMGlrS5cdoFKIg=";
+  };
+
+  cargoPatches = [
+    # https://gitlab.com/floers/calendar-stuff/karlender/-/merge_requests/43
+    # Remove it when it is merged and released.
+    (fetchpatch {
+      url = "https://gitlab.com/floers/calendar-stuff/karlender/-/commit/ce50c68323a834a0ee2cbff88ed285a971dfd91e.patch";
+      hash = "sha256-0hGgJPwnSNGTO/eiooQkQlBJ4brbaWVKRY6MT1PZApg=";
+    })
+    (fetchpatch {
+      url = "https://gitlab.com/floers/calendar-stuff/karlender/-/commit/98282a07aa92adeadeb5cff4fae2f32b73468357.patch";
+      hash = "sha256-nfbFdRYGmBzhaNwqwc8qXE9zCScT4Ra9bAth3i0q+qM=";
+    })
```

Comment:
I only added this commit because the CI on GitLab kept failing. Just now I realized these are compile warnings/hints treated as errors. I think for simplicity we don't need to fetch that patch as it still builds in nixpkgs (and one of those lints wasn't fixed before, if you want to keep it please update).